packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1064,13 +1064,13 @@ mixin WidgetInspectorService {
 
     _registerBoolServiceExtension(
       name: WidgetInspectorServiceExtensions.show.name,
-      getter: () async => WidgetsApp.debugShowWidgetInspectorOverride,
+      getter: () async => WidgetsApp.debugShowWidgetInspectorOverrideNotifier.value,
       setter: (bool value) {
-        if (WidgetsApp.debugShowWidgetInspectorOverride == value) {
+        if (WidgetsApp.debugShowWidgetInspectorOverrideNotifier.value == value) {
```

Comment:
nit: you can remove this check for whether the value is unchanged as ValueNotifier now handles that for you.